ssl/time.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <errno.h>
+#include <openssl/err.h>
+#include "internal/time.h"
+
+OSSL_TIME ossl_time_now(void)
+{
+#if defined(_WIN32)
+    SYSTEMTIME st;
+    union {
+        unsigned __int64 ul;
+        FILETIME ft;
+    } now;
+
+    GetSystemTime(&st);
+    SystemTimeToFileTime(&st, &now.ft);
+    /* re-bias to 1/1/1970 */
+# ifdef  __MINGW32__
+    now.ul -= 116444736000000000ULL;
+# else
+    /* *INDENT-OFF* */
```

Comment:
??? Why is this necessary?

Reply:
No idea, this was just copied wholesale from `d1_lib.c`.  The duplicate we have in `crypto/bio/bss_dgram.c` doesn't include these directives.

Reply:
There must have been a block of code that was formaterad in a special way for readability at some point in history...

The way this looks now, though, these indentation directives (that's what these comments are) do nothing of value. Remove them.

Reply:
Gone.